src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,37 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
```

Comment:
what is the purpose of this fragment?

Reply:
I think it's like small fail with pull request, because I haven't got  <>Put the card here</> on the fourth row  in VScode

Reply:
you don't need to use `<>` here because you have only one div inside (className="card")